src/amber/cli/templates/app/src/views/home/index.slang.ecr

```diff
@@ -1,8 +1,9 @@
-div#logo.col-sm-6
-div.col-sm-6
-  h2 = t "welcome_to_amber"
-  p Thank you for trying out the Amber Framework.  We are working hard to provide a super fast and reliable framework that provides all the productivity tools you are used to without sacrificing the speed.
-  div.list-group
-    a.list-group-item.list-group-item-action target="_blank" href="https://docs.amberframework.org" Getting Started with Amber Framework
-    a.list-group-item.list-group-item-action target="_blank" href="https://github.com/veelenga/awesome-crystal" List of Awesome Crystal projects and shards
-    a.list-group-item.list-group-item-action target="_blank" href="https://crystalshards.xyz" What's hot in Crystal right now
+div.row
+  div#logo.col-sm-6
```

Comment:
I think the `.col-sm-6` class isn't needed here

With `.col-sm-6` class:

![screenshot_20180411_102953](https://user-images.githubusercontent.com/3067335/38626921-47181c08-3d73-11e8-8b51-da7bbbefb150.png)


Without `.col-sm-6` class:

![screenshot_20180411_103036](https://user-images.githubusercontent.com/3067335/38626950-607270b8-3d73-11e8-83dd-9ba2417c85f9.png)

Reply:
oh. I forgot pushing my codes